__tests__/e2e/wallets.spec.ts

```diff
@@ -31,4 +31,18 @@ describe('wallets', () => {
       // created_at: '2021-10-08T02:33:20.732Z',
     });
   });
+
+  it('get wallet token-continent count ', async () => {
+    const response = await supertest(app).get(
+      '/wallets/token-region-count/eecdf253-05b6-419a-8425-416a3e5fc9a0',
+    );
+    expect(response.status).toBe(200);
+    expect(response.body.walletStatistics).toBeInstanceOf(Array);
+    expect(response.body.walletStatistics).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ continent: 'Africa' }),
+        expect.objectContaining({ continent: 'North America' }),
+      ]),
+    );
+  });
```

Comment:
I think GET /wallets/:walletId/token-region-count is a better url if this, right? 
according to convention of restful, https://en.wikipedia.org/wiki/Representational_state_transfer the resource is representing: wallet resource [id] 's token count

Reply:
The hierarchy is more reasonable I think

Reply:
yep URL is more structured as per your way